ui/pages/v2/Automations.tsx

```diff
@@ -10,10 +10,9 @@ type Props = {
 
 function Automations({ className }: Props) {
   const { data: automations, error, isLoading } = useListAutomations();
-
   return (
     <Page error={error} loading={isLoading} className={className}>
```

Comment:
this should maybe still be something like?
```suggestion
    <Page error={error || automations?.errors} loading={isLoading} className={className}>
```